systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -992,7 +992,7 @@ void testRegenerateCertExternalAddressChange() {
                     new GenericKafkaListenerBuilder()
                             .withName("external")
                             .withPort(9094)
-                            .withType(KafkaListenerType.NODEPORT)
+                            .withType(KafkaListenerType.LOADBALANCER)
```

Comment:
Why this caused issues?

Reply:
There was problem with certificate difference between `NODEPORT` and `LOADBALANCER` -> and also the original test had external listener `LOADBALANCER` as the tag shows. The difference in certificate was in several characters.

Reply:
The nodeport doesn't have any hostnames in cerificates so the certificates are same like without the nodeport. They are different for the loadbalancer.